cmd/capdctl/main.go

```diff
@@ -99,23 +97,67 @@ func main() {
 
 	switch os.Args[1] {
 	case "setup":
-		setup.Parse(os.Args[2:])
-		makeManagementCluster(*managementClusterName, *version, *capdImage, *capiImage)
+		if err := setup.Parse(os.Args[2:]); err != nil {
+			fmt.Printf("%+v\n", err)
+			os.Exit(1)
```

Comment:
klog.Fatal?

Reply:
i'm not actually sure why klog is in this file. If we need a logger we will use klogr. I'm going to play around with ripping out klog here and see why it exists in the first place.